contentcuration/contentcuration/utils/celery/tasks.py

```diff
@@ -66,6 +72,17 @@ def get_task_model(ref, task_id):
     return ref.backend.TaskModel.objects.get_task(task_id)
 
 
+def generate_task_signature(task_name, task_kwargs=None, channel_id=None):
+    """
+    :return: A hex string, md5
+    """
+    md5 = hashlib.md5()
+    md5.update(task_name.encode('utf-8'))
+    md5.update(str(task_kwargs or '').encode('utf-8'))
```

Comment:
My first thought here was that `task_kwargs` would be a dict so this wouldn't have stable key ordering, but I see that later we are passing the pre-encoded kwargs to this. Could add to the doc string clarifying that `task_kwargs` should already be a string?